src/cli/restore/sharepoint.go

```diff
@@ -64,6 +66,11 @@ func addSharePointCommands(cmd *cobra.Command) *cobra.Command {
 			utils.LibraryItemFN, nil,
 			"Restore library items by file name or ID")
 
+		fs.StringSliceVar(
```

Comment:
Halfway there: you're missing the StringSliceVar for `listPaths`.

Incidentally, line 60 is a bug and should be `&libraryPaths`

Reply:
Updates within [4a311ac](https://github.com/alcionai/corso/pull/1972/commits/4a311ac7405e7197bddf40cc32ce3de5110b216c)